code/framework/object/src/main/java/io/cattle/platform/object/process/ObjectProcessManager.java

```diff
@@ -25,4 +25,8 @@
 
     ExitReason executeProcess(String processName, Object resource, Map<String, Object> data);
 
+    void scheduleProcessInstanceAndPublishEvent(String processName, Object resource, Map<String, Object> data);
```

Comment:
I don't know that either name is clear to me.
`scheduleProcessInstanceAsync` any better?